web/src/features/fireWeather/components/maps/stationAccuracy.ts

```diff
@@ -0,0 +1,127 @@
+export interface StationMetrics {
+  observations: {
+    temperature: number
+    relative_humidity: number
+  } | null
+  forecasts: {
+    temperature: number
+    relative_humidity: number
+  } | null
+}
+
+export interface ColorResult {
+  temperature: string
+  relative_humidity: string
+}
+export const noDataColor = '#000000'
+export const neutralColor = '#DFDEDB'
+export const neutralIndex = 3
+
+export const rhColorScale = [
+  '#07A059',
+  '#3BAC48',
+  '#82C064',
+  neutralColor,
+  '#FCCE89',
+  '#F4A036',
+  '#ED8001'
+]
+export const tempColorScale = [
+  '#720505',
+  '#D05050',
+  '#F79191',
+  neutralColor,
+  '#60D3F8',
+  '#38B0F8',
+  '#2F80EE'
+]
+
+export const windColorScale = [
+  '#460270',
+  '#BC69EF',
+  '#D6B2ED',
+  neutralColor,
+  '#7AD3CE',
+  '#17B8A7',
+  '#089E83'
+]
+
+/**
+ *  Calculates the percentage difference between observed and forecasted station metrics.
+ *  Uses increments of 3% to determine color code in the associated color arrays.
```

Comment:
I thought the increments of 3% only applied to RH. It was my understanding that temperature should have increments of 2 degrees?

Reply:
oh duh, this ticket is only for RH.
Anyway, the code that is in here for temperature is wrong - shouldn't be using %age difference, should be calculating absolute degrees differentials

Reply:
Yep will update that when adding temperature. I'll note this on the next ticket.

Reply:
https://app.zenhub.com/workspaces/wildfire-predictive-services-5e321393e038fba5bbe203b8/issues/bcgov/wps/879